README.md

```diff
@@ -1,8 +1,65 @@
-# Microverse first repo
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-My first repo of microverse
+# Hello Microverse
 
-## Author
-Michael Hirpo
-mikaelhirpo@gmail.com
+> Description the project.
 
+
+## Built With
+
+- Html
+- css
+
+
+
+
+## Getting Started
+
+**git clone https://github.com/mikihirpo/microverse-first.git**
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+
```

Comment:
- [ ] Please add your name here because you are the author of this repository. :+1: